nixos/modules/services/search/meilisearch.nix

```diff
@@ -64,6 +58,15 @@ in
       type = with types; nullOr path;
     };
 
+    datadir = mkOption {
```

Comment:
I think it should be `dataDir` by [convention](https://search.nixos.org/options?channel=24.11&size=50&sort=relevance&type=packages&query=datadir).